coding-concepts.js

```diff
@@ -6,54 +6,54 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//  console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: indigo will be pushed into colors making the value number of the array 5.
+// b) Verify and explain: 5 is wha i got after verifying which is correct now there are 5 vales in the array when the indigo is added.
 
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
-// console.log(cohort.length)
+// const cohort = "LEARN 2022"
+//  console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the answer would be the length of the cohort which is 9
+// b) Verify and explain: i got back 10 which most likely the space was also counted making the length 10 
 
 
 // --------------------2) What will this log?
-
-const greeting = "Hello World!"
+ 
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: greeting is the variable for hello world, i will most likely get hello world in my terminal 
+// b) Verify and explain: the answer is the letter o because the value 4 is the o value. something  I totally ignored.
 
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: index will be placed at the begining of the array. 
+// b) Verify and explain: I got ruby only in the terminal, I suppose the value 1 is "rubby" which is the only explaination. 
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
+// const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: string will be written in uppercase 
+// b) Verify and explain: could not verify becuase output says;weekDays.toUpperCase is not a function. I tried to fix the code but couldnt find out. I think the output after fixing the code to captilize the string. 
```

Comment:
This one is super tricky!  The method .toUpperCase only works on the string data type and not on arrays so this code will result in an error.  To capitalize both values, some sort of iteration would be needed here.

Reply:
this drove me crazyyy! ill try it for sure bc i would love to see it work.